webrtc.html

```diff
@@ -11325,31 +11325,39 @@ <h2>
                 <dfn data-idl="">setCodecPreferences</dfn>
               </dt>
               <dd>
-		<div id="setcodecpreferences-receive">
+    <div id="setcodecpreferences-send-and-receive">
                 <p>
                   The {{setCodecPreferences}} method overrides the default
-                  receive codec preferences used by the <a>user agent</a>. When
+                  codec preferences used by the <a>user agent</a>. When
```

Comment:
Can we add a note somewhere to explain how setCodecPreferences works differently in the offerer vs answerer role? 

For example search for "Looking at setCodecPreferences as a sender-control leads to surprises" in https://blog.mozilla.org/webrtc/cross-browser-support-for-choosing-webrtc-codecs/

Reply:
Notes clarified, including that the answer defaults to offer order